samples/texture_immutable.html

```diff
@@ -15,7 +15,7 @@
 
 <body>
     <div id="info">WebGL 2 Samples - texture_immutable</div>
-    <p id="description">Immutable texture refers to the texture's allocation, not the content of the texture. Therefore, new pixel data can be uploaded, but the size of the texture storage can't change.</p>
+    <p id="description">Immutable texture refers to the texture's allocation, not the content of the texture. Therefore, new pixel data can be uploaded, but the size of the texture storage can't change. (currently not working for texture 3D)</p>
```

Comment:
We are probably better off just mentioning this in the README.md so we don't forget to update here later.